templates/blog-post.html

```diff
@@ -8,6 +8,16 @@
 {% block header %}{{ this.parent.title }}{% endblock %}
 {% block body %}
   <h2 class="mb-0">{{ this.title }}</h2>
+  {% if this.series|length %}
+  <div class="card bg-light">
+    <div class="card-body p-2">
+    <p class="meta my-0"><span class="mr-2"><strong>This blog is part of the series :</strong></span>
+       <a href="/blog/series/{{this.series}}/">{{ this.series }}</a>
+    </p>
+    </div>
+  </div>
+ {% endif %}
+
```

Comment:
It would be great if you could show all posts in the series either at the bottom of the post or to the side (like a table of contents).